### PR TITLE
Table Store Release 0.3.0

### PR DESCRIPTION
**Pull request by JingsongLi**

### CODE REVIEW

File: _posts/2023-01-15-release-table-store-0.3.0.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+[Apache Flink Table Store](https://github.com/apache/flink-table-store) (0.3.0).
+
+We highly recommend all users upgrade to Flink Table Store 0.3.0. 0.3.0 completed 150+ issues, which were completed by nearly 30 contributors.
+
+Please check out the full [documentation]({{site.DOCS_BASE_URL}}flink-table-store-docs-release-0.3/) for detailed information and user guides.
+
+<br/>
+
+Flink Table Store 0.3 completes many exciting features, Enhance its ability as a data lake and greatly
+improve the availability of stream pipeline. Some important features are described below.
```
Comment (carp84) [inline _posts/2023-01-15-release-table-store-0.3.0.md]:
```suggestion
Flink Table Store 0.3 completes many exciting features, enhances its ability as a data lake storage and greatly
improves the availability of its stream pipeline. Some important features are described below.
```

File: _posts/2023-01-15-release-table-store-0.3.0.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+    --table <table-name>
+```
+
+## Aggregation Table
+
+Sometimes users only care about aggregated results. The
+[aggregation merge engine]({{site.DOCS_BASE_URL}}flink-table-store-docs-release-0.3/docs/features/table-types/#aggregation)
+aggregates each value field with the latest data one by one under the same primary key according to the aggregate function.
+
+Each field not part of the primary keys must be given an aggregate function, specified by the
```
Comment (carp84) [inline _posts/2023-01-15-release-table-store-0.3.0.md]:
```suggestion
Each field that is not part of the primary keys must be given an aggregate function, specified by the
```

**Review by carp84 — APPROVED** [review]
+1, just some minor wording refinement suggestions.

Side note: with new features added, maybe it worth updating the [quick-start](https://nightlies.apache.org/flink/flink-table-store-docs-master/docs/try-table-store/quick-start/) document to include more scenarios such as time traveling?

File: _posts/2023-01-15-release-table-store-0.3.0.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+the new schema when it is read.
+
+For example:
+
+```
+CREATE TABLE T (i INT, j INT);
+
+INSERT INTO T (1, 1);
+
+ALTER TABLE T RENAME COLUMN i to a;
```
Comment (wuchong) [inline _posts/2023-01-15-release-table-store-0.3.0.md]: What do you think about adding `ADD COLUMN` in the example that users might be more interested in?

File: _posts/2023-01-15-release-table-store-0.3.0.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+3 rows in set
+*/
+```
+
+## Ecosystem
+
+Flink Table Store continues to strengthen its ecosystem and gradually gets through the reading and writing
+of all engines. Each engine below 0.3 has been enhanced.
+
+- Spark write has been supported. But `INSERT OVERWRITE` and stream write are still supported.
```
Comment (wuchong) [inline _posts/2023-01-15-release-table-store-0.3.0.md]: Are `INSERT OVERWRITE` and stream write supported?

File: _posts/2023-01-15-release-table-store-0.3.0.md [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+- [S3]({{site.DOCS_BASE_URL}}flink-table-store-docs-release-0.3/docs/filesystems/s3/) and
+  [OSS]({{site.DOCS_BASE_URL}}flink-table-store-docs-release-0.3/docs/filesystems/oss/)
+  are supported by all computing engines.
+- Hive 3.1 are supported.
+- Trino High versions (JDK 1.17) are supported.
```
Comment (wuchong) [inline _posts/2023-01-15-release-table-store-0.3.0.md]: Why is the "High" capitalized? Is "Trino High" refers to specific terminology? Would be better to clarify what more versions are supported.

**Review by wuchong — APPROVED** [review]
+1 after my minor comments are addressed.

File: _posts/2023-01-15-release-table-store-0.3.0.md (hunk 4 above)
Comment (JingsongLi) [reply _posts/2023-01-15-release-table-store-0.3.0.md]: NOT, I forgot add `NOT`...

**Comment by JingsongLi** [comment]
> 

Thanks for your suggestion.
I think we should keep quick-start simple, but we do need some good links on the documents to expose our new features.

**Comment by JingsongLi** [comment]
> +1, just some minor wording refinement suggestions.
> 
> Side note: with new features added, maybe it worth updating the [quick-start](https://nightlies.apache.org/flink/flink-table-store-docs-master/docs/try-table-store/quick-start/) document to include more scenarios such as time traveling?

Thanks for your suggestion.
I think we should keep quick-start simple, but we do need some good links on the documents to expose our new features.

File: _posts/2023-01-15-release-table-store-0.3.0.md (hunk 3 above)
Comment (JingsongLi) [reply _posts/2023-01-15-release-table-store-0.3.0.md]: Sounds good. I will add `ADD`

File: _posts/2023-01-15-release-table-store-0.3.0.md (hunk 5 above)
Comment (JingsongLi) [reply _posts/2023-01-15-release-table-store-0.3.0.md]: I will change this to JDK 1.17 versions

Comment (JingsongLi) [reply _posts/2023-01-15-release-table-store-0.3.0.md]: Currently, the api has not changed, so we can support the new version

**Pull request merged**